README.md

```diff
@@ -1,61 +1,102 @@
 # [Hackster](./README.md) &middot; [![GitHub license]](./LICENSE) ![CI](https://github.com/hackthebox/hackster/actions/workflows/test.yaml/badge.svg) [![codecov](https://codecov.io/gh/hackthebox/Hackster/branch/main/graph/badge.svg?token=DSQFU4YP2W)](https://codecov.io/gh/hackthebox/Hackster)
 
+Welcome to the Hackster project! Its goal is to enhance the user experience for HTB's community members, and therefore
+it is always going to be a work in progress. We've been inspired by the fantastic work of other projects,
+particularly [Noahbot](https://github.com/HeckerBirb/NoahBot), and we're excited to contribute our own ideas and
+features to the broader community.
+
 <!-- Table of Contents -->
 
-- [Installation](#installation)
-- [Environment Variables](#environment-variables)
-- [Usage](#usage)
+## Table of Contents
+
+- [Features](#features)
+- [Getting Started](#getting-started)
+- [Contributing](#contributing)
 - [License](#license)
+- [Code of Conduct & Security](#code-of-conduct--security)
+- [Special Thanks](#special-thanks)
+- [Questions & Support](#questions--support)
+- [Contributors](#contributors)
 
-## Installation
+## Features
 
-The first step will be to clone the repo
+- Message moderation: filter, delete, or flag inappropriate content.
+- User management: warn, mute, kick, or ban users based on customizable rules.
+- CTF Events management: create and manage channels, roles, and permissions for CTF Events.
+- And much more!
+- And much more!
 
-```shell
-git clone https://github.com/hackthebox/hackster.git
-```
+## Getting Started
 
-The requirements are:
+To set up and deploy the Discord bot, follow these steps:
 
-* [Python] and [Poetry]
+1. The first step will be to clone the repo
+   ```shell
+   git clone https://github.com/hackthebox/hackster.git
+   ```
+   The requirements are:
+    * [Python](https://www.python.org/)
+    * [Poetry](https://python-poetry.org/)
 
-1. Install the dependencies
+2. Install the dependencies
    ```shell
    poetry install
    ```
 
-## Environment Variables
+3. add the following environment variables.
 
-To run this project, you will need to add the following environment variables.
+   | Variable       | Description                | Default    |
+                                                                                                                                                                           |----------------|----------------------------|------------|
+   | BOT_NAME       | The name of the bot        | "Hackster" |
+   | BOT_TOKEN      | The token of the bot       | *Required  |
+   | CHANNEL_DEVLOG | The devlog channel id      | 0          |
+   | DEBUG          | Toggles debug mode         | False      |
+   | DEV_GUILD_IDS  | The dev servers of the bot | []         |
+   | GUILD_IDS      | The servers of the bot     | *Required  |
```

Comment:
The table seems to be broken
![image](https://user-images.githubusercontent.com/8687447/235884530-4eb2eb7f-e15e-4b57-ade9-d8aac6b2f15c.png)